inc/exhibits-detail.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * The sub-template used by page-exhibits-home.php, page-exhibits-feed.php,
+ * category-rotch-library.php and category-maihaugen-gallery.php for
+ * displaying Current Exhibits.
+ *
+ * @package MIT_Libraries_Child
+ * @since 2.0.0
+ */
+
+?>
+
+
+			<div class="exhibits-feed">
+				<div class="entry-categories">
+					<div class="entry-cats-list">
+
+							<a href="/exhibits/<?php echo esc_attr( str_replace( ' ','-',get_field( 'location' ) ) ); ?>"><span class="category-bg"><span class="category-init"><?php echo esc_attr( substr( get_field( 'location' ),0,1 ) ); ?></span></span><span class="cat-name"><?php the_field( 'location' ) ?> Exhibit</span></a>
```

Comment:
Is there a way to write this line so it isn't hard-coded to the Exhibits site? Is that available via a Wordpress function in some way?